src/main/java/com/github/dockerjava/core/RemoteApiVersion.java

```diff
@@ -15,7 +16,7 @@
  *
  * @author Marcus Thiesen
  */
-public class RemoteApiVersion {
+public class RemoteApiVersion implements Serializable {
```

Comment:
Test ate my head, will do of course https://github.com/docker-java/docker-java/pull/377#issuecomment-158441711